src/test/java/org/verapdf/wcag/algorithms/semanticalgorithms/ChunksMergeUtilsTest.java

```diff
@@ -0,0 +1,243 @@
+package org.verapdf.wcag.algorithms.semanticalgorithms;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.verapdf.wcag.algorithms.entities.SemanticTextChunk;
+import org.verapdf.wcag.algorithms.semanticalgorithms.utils.ChunksMergeUtils;
+
+public class ChunksMergeUtilsTest {
+
+    @Test
+    public void toChunkMergeProbabilityDifferentFontsTest() {
+        SemanticTextChunk first = new SemanticTextChunk();
```

Comment:
will be better to add private method which creates SemanticTextChunk to avoid duplicated code